### DISCUSSION TITLE
AutoML Fails when reaching LinearSVR Model

### DISCUSSION THREAD

**Discussion by thepranj** [vertica/VerticaPy#1219] [Potential bug] [answered]
![image](https://github.com/vertica/VerticaPy/assets/25474030/366dfd78-0753-4b87-8913-75e3b8c3c5e2)

AutoML errors out when we reach the LinearSVR Model Type. TypeError: LinearSVR.__init__() got an unexpected keyword argument 'fit_intercept'. Other model types work fine. 

How to reproduce: I called AutoML.fit() on the Spotify example dataset, predicting popularity (continuous variable) using all numerical predictors. I did not test on other datasets.

**Comment by oualib** [comment, marked as answer]
Corrected in https://github.com/vertica/VerticaPy/pull/1239
